### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by georgedigkas**

### CODE REVIEW

**Comment by Bridgerz** [comment]
I like some of these changes! @georgedigkas can you merge main into this?

**Comment by georgedigkas** [comment]
> I like some of these changes! @georgedigkas can you merge main into this?

done!

File: contracts/BridgeVault.sol [hunk 1]
```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```
Comment (Bridgerz) [inline contracts/BridgeVault.sol:5]: we still need this import correct?

File: contracts/BridgeVault.sol [hunk 2]
```diff
@@ -50,7 +49,9 @@ contract BridgeVault is Ownable, IBridgeVault {
         wETH.withdraw(amount);
 
         // Transfer the unwrapped ETH to the target address
-        targetAddress.transfer(amount);
+        // Send ether and specify a custom gas amount
+        (bool success, ) = targetAddress.call{value: amount/**, gas: 5000*/}("");
```
Comment (Bridgerz) [inline contracts/BridgeVault.sol]: there's no need to check for transfer success. it either succeeds or fails and reverts the TX. 

lets leave what we have before

File: contracts/SuiBridge.sol [hunk 3]
```diff
@@ -3,7 +3,6 @@ pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts-upgradeable/utils/PausableUpgradeable.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```
Comment (Bridgerz) [inline contracts/SuiBridge.sol]: we still need this correct?

File: contracts/BridgeVault.sol (hunk 1 above)
Comment (georgedigkas) [reply contracts/BridgeVault.sol:5]: You are right, we need/use it, but it is inherited from: `import "./interfaces/IWETH9.sol";`

File: contracts/SuiBridge.sol (hunk 3 above)
Comment (georgedigkas) [reply contracts/SuiBridge.sol]: You are right, we need/use it, but it is inherited from: `import "./interfaces/IWETH9.sol";`

File: contracts/BridgeCommittee.sol [hunk 4]
```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import "./interfaces/IBridgeCommittee.sol";
```
Comment (Bridgerz) [inline contracts/BridgeCommittee.sol:5]: We still need this interface too.

File: contracts/BridgeVault.sol (hunk 1 above)
Comment (Bridgerz) [reply contracts/BridgeVault.sol:5]: Good point. Let's remove it.

File: contracts/BridgeVault.sol [hunk 5]
```diff
@@ -50,7 +49,8 @@ contract BridgeVault is Ownable, IBridgeVault {
         wETH.withdraw(amount);
 
         // Transfer the unwrapped ETH to the target address
-        targetAddress.transfer(amount);
+        // Send ether and specify a custom gas amount
+        targetAddress.call{value: amount/**, gas: 5000*/}("");
```
Comment (Bridgerz) [inline contracts/BridgeVault.sol]: Why this change?

Comment (georgedigkas) [reply contracts/BridgeVault.sol]: https://consensys.github.io/smart-contract-best-practices/development-recommendations/general/external-calls/#dont-use-transfer-or-send

`transfer` and `send` calls are no longer considered best practice
Having said that, I do not have a strong opinion I can change it back to `transfer`

File: contracts/BridgeCommittee.sol (hunk 4 above)
Comment (georgedigkas) [reply contracts/BridgeCommittee.sol:5]: more or less the same as the others, it is inherited from `import "./utils/CommitteeUpgradeable.sol";`

File: contracts/BridgeVault.sol (hunk 5 above)
Comment (Bridgerz) [reply contracts/BridgeVault.sol]: Ok lets go with consensys on this. can you remove the gas comment and change the inline comment (this forwards all available gas correct?
